daemon/src/peer/announcement.rs

```diff
@@ -75,31 +75,30 @@ async fn build<S>(peer: &Peer<S>) -> Result<Updates, Error>
 where
     S: Clone + Signer,
 {
-    let mut list: Updates = HashSet::new();
-
-    match state::list_projects(peer).await {
+    let identities = match state::list_identities(peer).await {
         // TODO(xla): We need to avoid the case where there is no owner yet for the peer api, there
         // should be machinery to kick off these routines only if our app state is ready for it.
-        Err(state::Error::Storage(librad::git::storage::Error::Config(_err))) => Ok(list),
-        Err(err) => Err(err.into()),
-        Ok(projects) => {
-            for project in &projects {
-                if let Some(refs) = state::list_owner_project_refs(peer, project.urn()).await? {
-                    for ((one_level, oid), category) in refs.iter_categorised() {
-                        list.insert((
-                            Urn {
-                                path: Some(RefLike::from(category).join(one_level.clone())),
-                                ..project.urn()
-                            },
-                            *oid,
-                        ));
-                    }
-                }
-            }
-
-            Ok(list)
-        },
+        Err(state::Error::Storage(_)) => Vec::new(),
```

Comment:
It's a less elaborate version of this removed line:
```rust
// TODO(xla): We need to avoid the case where there is no owner yet for the peer api, there
// should be machinery to kick off these routines only if our app state is ready for it.
 Err(state::Error::Storage(librad::git::storage::Error::Config(_err))) => Ok(list),
```
I have no idea what does it mean, so I've kept both the logic and the comment.

Reply:
These are not equivalent, the original ignores exactly one possible storage error while the rest is surfaced. With the new version all storage errors are masked.

Reply:
Good catch, I haven't noticed that it's an enum variant! Fixed.